install/audit-logs.tf

```diff
@@ -0,0 +1,95 @@
+resource "aws_s3_bucket" "taskcluster_audit_logs" {
+  bucket = "taskcluster-staging-logs"
+}
+
+resource "aws_kinesis_stream" "taskcluster_audit_logs" {
+  name             = "taskcluster-staging-audit-logs"
```

Comment:
Is this only going to be ran in staging or does this need to have a variable for `staging` vs. `prod`?

Reply:
This one is just staging. We actually have two separate configurations for prod/staging.